requirements.txt

```diff
@@ -14,4 +14,4 @@ v3io-frames>=0.10.2,<1
 # which breaks v3io-frames
 protobuf>=3.20.1,!=3.20.2,<4
 v3iofs~=0.1.9
-xxhash>=1
+xxhash>=1
```

Comment:
Please don't change this line.

Reply:
the tests are failing without this requirement as you can see here https://github.com/mlrun/storey/pull/384/commits/650afa6bf2b7414cfd58c711e4e1433ca293bf74

Reply:
I didn't changed any thing- you can see that the lines are exactly the same